test/scripts/e2e-kind.sh

```diff
@@ -58,6 +79,10 @@ case "$SHARD" in
 		# all tests that have P as the sixth letter after the N
 		GINKGO_ARGS="${GINKGO_ARGS} "'--ginkgo.focus=\[sig-network\]\s[Nn].{6}[Pp].*$'
 		;;
+	shard-ipv4)
+		IPV4_ONLY_TESTS=$(echo "${IPV4_ONLY_TESTS}" | sed -e '/^\($\|#\)/d' -e 's/ /\\s/g' | tr '\n' '|' | sed -e 's/|$//')
```

Comment:
should probably have a bash function to do that `sed|tr|sed`

Reply:
OK, will do.